portal-web/test/functional/com/liferay/portalweb/tests/enduser/publications/ManagePublications.testcase

```diff
@@ -10,11 +10,6 @@ definition {
 
 		User.firstLoginPG();
 
-		ApplicationsMenu.gotoPortlet(
-			category = "Publications",
-			panel = "Applications",
-			portlet = "Publications");
-
```

Comment:
Good call, it doesn't add much clarity when we have this step at the testcase level.